src/hooks/useGraphSyncStatus.ts

```diff
@@ -59,7 +59,7 @@ export function useGraphSyncStatus(): UseGraphSyncStatus {
   const { block, web3Loading } = useWeb3()
   const [blockGraph, setBlockGraph] = useState<number>()
   const [blockHead, setBlockHead] = useState<number>()
-  const [isGraphSynced, setIsGraphSynced] = useState(true)
+  const [isGraphSynced, setIsGraphSynced] = useState(false)
```

Comment:
won't this result in the faulty graph sync status showing up all the time first, and then disappearing? Should be the other way around, should only come up when there is an error detected, not by default

Reply:
no, this was just for testing and forgot to change it, good catch